### DISCUSSION TITLE
Custom Font in @storybook/theming Not Working

### DISCUSSION THREAD

**Discussion by jpmarra** [storybookjs/storybook#12032] [Help]
Hey folks! Looking to add some brand flavor to our Storybook, and I'm unable to configure a custom font inside of the `create` fn.

I have the font loaded in and defined in our css for our components. Is there somewhere else I need to define the font face in order for it to be available for use within the `fontBase` property of `create`? Any help would be appreciated!

Here's what I have at the moment:

```javascript
// viceTheme.js

import { create } from '@storybook/theming/create';

export default create({
  base: 'light',
  colorPrimary: 'black',
  colorSecondary: 'white',

  // UI
  appBg: 'black',
  appContentBg: 'white',
  appBorderColor: 'white',
  appBorderRadius: 0,

  // Typography
  fontBase: '"Source Code Pro", sans-serif', // <-- DOESN'T WORK CURRENTLY
  fontCode: 'monospace',

  // Text colors
  textColor: 'white',
  textInverseColor: 'black',

  // Toolbar default and active colors
  barTextColor: '#ccc',
  barSelectedColor: 'white',
  barBg: 'black',

  // Form colors
  inputBg: 'black',
  inputBorder: 'white',
  inputTextColor: 'white',
  inputBorderRadius: 0,

  brandTitle: 'VICE',
  brandUrl: 'https://vice.com',
  brandImage: 'https://placehold.it/350x150',
});
```

**Comment by shilman** [comment]
@domyen @ndelangen ?

**Comment by ndelangen** [comment]
That would be a bug, I think... the ThemeVar property is used / converted here:
https://github.com/storybookjs/storybook/blob/a18f7e1dedab1a4b36f570804a7fa1807b60f625/lib/theming/src/convert.ts#L120-L122

But it might just not be used in some places (styled components)?

Noteworthy seems to be this gem:
https://github.com/storybookjs/storybook/blob/99b69566b1a30f332afe9bff9526df2d99de0ce2/addons/docs/src/blocks/Wrapper.tsx#L3-L5

**Reply by thesobercoder** [reply]
I'm also stuck with the same issue. Please specify a workaround.
